src/libcore/fmt/mod.rs

```diff
@@ -609,10 +609,15 @@ pub trait Debug {
 /// println!("The origin is: {}", origin);
 /// ```
 #[rustc_on_unimplemented(
+    on(
+        _Self="std::path::Path",
+        label="`{Self}` cannot be formatted with the default formatter, call `.display()` on it",
+        note="you need to call `.display()` or `.to_string_lossy()` for safely printing paths as \
```

Comment:
```suggestion
        note="you need to call `.display()` or `.to_string_lossy()` for safely printing paths, as \
```
"as they may [...]" is a dependent clause and should be set off by a comma.